backend/src/routes/projectRouter.ts

```diff
@@ -135,15 +135,16 @@ projects.get("/v1/project_lookup/:name", async (req, res, next) => {
   }
 });
 //search projects by name and description
-projects.get("/v1/project_search/:search", async (req, res, next) => {
-  try {
-    const project = await projectController.searchProjects(
-      req.params["search"]
-    );
-    res.json(project);
-  } catch (err) {
-    next(err);
-  }
+projects.get("/v1/project_search", async (req, res, next) => {
+  if (req.query["search"])
```

Comment:
this condition needs an else branch, otherwise the request times out. if the query parameter is missing, we should return an error to indicate that to the caller, something like:
```
response 400
{
  "errors": [
    "missing_search"
  ]
}
```

so the frontend can handle it appropriately